SECURITY.md

```diff
@@ -0,0 +1,17 @@
+# Security Policy
+
+## Supported Versions
+
+| Version     | Supported          | Status                | Comments                             |
+| ----------- | ------------------ | --------------------- | ------------------------------------ |
+| 1.1.x       | :white_check_mark: | Active Development    |                                      |
+| <1.0.x      | :x:                |                       |                                      |
+
+## Reporting a Vulnerability
+
+Email us at [support@angularjsmaterial.dev](mailto:support@angularjsmaterial.dev) to report any potential security
```

Comment:
Who owns this? AFAIK this should be security@angularjs.org for now.

Reply:
Mentioned in the end of the description:
> support@angularjsmaterial.dev goes directly to me and has specific importance filters set up for it in Gmail.

I can do security@angularjs.org, I just don't have any access to it or any idea if the information would ever get to me.

Reply:
Yeah, definitely security@angularjs.org ; we'll let you know if anything comes in (nothing has so far)

Reply:
Updated.